### PR TITLE
update hosted env to use newer openmm-torch version

### PR DESCRIPTION
**Pull request by mikemhenry**

### CODE REVIEW

**Comment by mikemhenry** [comment]
Once the build is green here https://github.com/conda-forge/openmm-torch-feedstock/commits/rc

We should be good to merge this in and update the hosted env, I want to wait till it hits the CDN so that people won't get a "version can't be found" error

**Comment by peastman** [comment]
We need to add the pytorch 1.13 builds first.

**Comment by mikemhenry** [comment]
PR for that here: https://github.com/conda-forge/openmm-torch-feedstock/pull/31

**Comment by peastman** [comment]
I tried to create an environment with the new file, but it fails with this error:

```
Encountered problems while solving:
  - package openmm-ml-1.0beta-pyh79ba5db_0 requires openmm-torch 1.0beta, but none of the providers can be installed
```

**Comment by peastman** [comment]
Possibly we should rebuild the openmm-torch package with the same version as before (1.0beta), just updating the build number.  There weren't any code changes in it, so there's no need for an updated version number.  That will maintain compatibility with the other beta packages.

**Comment by mikemhenry** [comment]
> Possibly we should rebuild the openmm-torch package with the same version as before (1.0beta), just updating the build number. There weren't any code changes in it, so there's no need for an updated version number. That will maintain compatibility with the other beta packages.

I agree, honestly, that is what I thought I was doing when I changed the version number. I will fix that with https://github.com/conda-forge/openmm-torch-feedstock/pull/31

File: devtools/hosted-envs/openmm-8-rc1-linux.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+channels:
+  - conda-forge/label/openmm_rc
+  - conda-forge/label/openmm-torch_rc
+  - conda-forge
+
+dependencies:
+  - openmm==8.0.0rc1
+  - openmm-ml==1.0rc1
+  - openmm-torch==1.0rc1
+  - nnpops==0.2
```
Comment (peastman) [inline devtools/hosted-envs/openmm-8-rc1-linux.yaml:12]: This should be 0.3.

**Comment by mikemhenry** [comment]
@peastman good catch! Fixed + uploaded new envs to my mmh channel

**Pull request merged**